webapp/channels/src/actions/admin_actions.jsx

```diff
@@ -550,6 +550,17 @@ export function upgradeToE0Status() {
     };
 }
 
+export function isAllowedToUpgradeToEnterprise() {
+    return async () => {
+        try {
+            await Client4.isAllowedToUpgradeToEnterprise();
+            return {};
+        } catch (error) {
+            return error;
```

Comment:
For consistency, could you make this return an object with a data field or an error field? You'll also need to update the code that calls it as well

```js
try {
    await Client4.isAllowedToUpgradeToEnterprise();
    return {data: true};
} catch (error) {
    return {error};
}
```